testbed/infrastructure.ts

```diff
@@ -41,5 +41,14 @@ export class Infrastructure extends cdk.Construct {
             minSize: 5,
             maxSize: 20
         });
+        
+        //service account used by tekton workflows.
+        const sa = this.cluster.addServiceAccount('test-executor', {
+            name: "test-executor",
+        });
+
+        sa.role.addManagedPolicy({
```

Comment:
any reason to not inline this?
```
this.cluster
  .addServiceAccount('test-executor', { name: 'test-executor' })
  .role.AddManagedPolicy({ managedPolicyArn: `arn:aws:iam::aws:policy/AdministratorAccess` })
```

I'm also noticing inconsistent use of ` ' and ". 

Take a look at https://blog.cloudboost.io/js-bytes-double-quote-vs-single-quote-vs-backtick-44e36c390dbd